features/float16array.yml

```
@@ -1,7 +1,7 @@
-draft_date: 2024-08-12
 name: Float16Array
-description: TODO
+description: Float16Array is a typed array of 16-bit floating point numbers in the platform byte order.
```

In https://github.com/web-platform-dx/web-features/pull/1628 it was suggested that we don't need this much detail.

```suggestion
description: Float16Array is a typed array of 16-bit floating point numbers.
```